omg/entities/elements.py

```diff
@@ -0,0 +1,35 @@
+import os
+import json
+from typing import Dict
+
+ELEMENTS_JSON_DIR = os.path.join(os.path.dirname(__file__), "Elements.JSON")
+
+
+class DotDict(dict):
+    """Convert dictionary to a format to call attributes with dot notation."""
+
+    def __getattr__(self, attr):
+        """Get dictionary item as attribute.Convert nested dictionaries to DotDict."""
+        value = self.get(attr)
```

Comment:
To be robust with dictionaries you would like to use `value = dict.get(key, None)` syntax. Then the dictionary will return a default (`None` in this case) and won't crash if the queried key does not exist. Otherwise, the code will crash with `KeyError`.